### DISCUSSION TITLE
[DOC- 1722] Overflowing Sidebar Content in Documentation

### DISCUSSION THREAD

**Discussion by uneknown** [vercel/community#3716] [Bug Report]
### Page to Investigate

https://vercel.com/docs/errors/application-error-codes

### Steps to Reproduce

The content within the right sidebar ("On this page") is overflowing.

![overflow](https://github.com/vercel/community/assets/111852433/a08b8074-5b00-4346-a9fe-fb001093f422)

By the way, is it possible to switch back to the previous version of the documentation? The new documentation is helpful for newcomers, but not ideal when trying to find specific information.

**Comment by swarnava** [comment]
Hi,

Is it possible to share us a screencast so we can take a closer look? Which OS / Device / Browser are you using to reproduce this?

Unfortunately, it's not possible to switch to old theme. Could you share more details about the trouble you faced while finding an information so I can pass to relevant team? Thanks!

**Reply by uneknown** [reply]
Operating Systems: Windows 10, Windows 11, MacOS Ventura, MacOS Sonoma
Browsers:
- Firefox 116.0.2
- Latest Chromium-based browsers (Opera, Chrome, Edge)
- Latest Safari

The issue remains consistent across all these operating system versions and web browsers. I'm currently pressed for time; I'll provide further details in a few hours.

**Reply by swarnava** [reply]
I could reproduce this and shared with my team to investigate. Thank you for reporting! If you are having trouble to find any specific article in our new Docs page, please let us know and we will forward that too.
